framework/utils/xmlUtils.py

```diff
@@ -206,16 +206,16 @@ def fixXmlText(msg):
     @ Out, msg, string, fixed string
   """
   #if not a string, pass it back through
-  if not isinstance(msg,basestring):
+  if not type(msg).__name__ in ['unicode','str']:
```

Comment:
Seems strange, they are in the same directory? How is xmlUtils found but not utils? How about try:

```python
from .utils.utils import isString
```

This way it should know to look for the utils submodule in the current module.